src/control/cmd/dmg/pool.go

```diff
@@ -562,6 +562,18 @@ func (cmd *PoolSetPropCmd) Execute(_ []string) error {
 		cmd.Args.Props.ToSet = []*control.PoolProperty{p}
 	}
 
+	for _, prop := range cmd.Args.Props.ToSet {
+		if prop.Name == "rf" {
+			return errors.New("can't set redundancy factor if pool is created.")
+		}
+		if prop.Name == "ec_pda" {
+			return errors.New("can't set EC performance domain affinity if pool is created.")
+		}
+		if prop.Name == "rp_pda" {
+			return errors.New("can't set RP performance domain affinity if pool is created.")
+		}
+	}
+
```

Comment:
OK for now, but I'm wondering if this should be defined in the pool property handler map instead, in the long term. Container props have a read-only flag, we could do something similar for pool props.

An aside, the message is a little awkward in English. Instead of "if pool is created," how about "on existing pool"?

Reply:
Sure, fixed comments, let's address Readonly in seperate PR.